docs/core/context.md

```diff
@@ -79,12 +63,12 @@ goes wrong. The pattern of usage for a Context is as follows:
 
 1. A process receives a Context `ctx` from its parent process, which provides information needed to
    perform the process.
-2. The `ctx.ms` is **cache wrapped**, i.e. a cached copy of the [multistore](./store.md#multistore) is made so that the process can make changes to the state as it executes, without changing the original`ctx.ms`. This is useful to protect the underlying multistore in case the changes need to be reverted at some point in the execution. 
+2. The `ctx.ms` is **cache wrapped**, i.e. a cached copy of the [multistore](./store.md#multistore) is made so that the process can make changes to the state as it executes, without changing the original`ctx.ms`. This is useful to protect the underlying multistore in case the changes need to be reverted at some point in the execution.
```

Comment:
```suggestion
2. The `ctx.ms` is **snapshot**, i.e. a [multistore](./store.md#multistore) branch is made so that the process can make changes to the state as it executes, without changing the original`ctx.ms`. This is useful to protect the underlying multistore in case the changes need to be reverted at some point in the execution.
```

Reply:
Cached wrapper still doesn't sound good. I suggest to use snapshot or branch.

Reply:
I noticed that "cache-wrapped" is use in multiple places. Maybe another PR to fix it consistently would be a better idea? (I can create an item in the docs issue)

Also, I'm not 100% comfortable with the "snapshot" word, imo it should be reserved for TM state sync.

Reply:
ok, let's do it in othe PR. I can have take a look into that.

Reply:
Done. I assigned you to the last item in https://github.com/cosmos/cosmos-sdk/issues/7659#issue-728568984